modules/core/src/main/scala/higherkindness/droste/basis.scala

```diff
@@ -26,6 +30,9 @@ object Embed extends FloatingBasisInstances[Embed] {
   def apply[F[_], R](implicit ev: Embed[F, R]): Embed[F, R] = ev
 }
 
+/**
+  * A `Project[F, R]`
```

Comment:
Might as well say the dual of Embed!

"Project[F, R] provides implicit evidence of a Coalgebra[F, R] for use in folds from structure `R`."